### PR TITLE
Added scorecard tests

### LINKED ISSUE #4: Add scorecard tests

### PR DESCRIPTION
**Pull request by yuriolisa**
Signed-off-by: Yuri Sa <yurimsa@gmail.com>
Close #4

### CODE REVIEW

**Comment by yuriolisa** [comment]
@pavolloffay

**Review by pavolloffay — APPROVED** [review]
approving, assuming the CI passes

File: .github/workflows/scorecard.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v3
+
+      - name: "install kuttl and kind"
```
Comment (iblancasa) [inline .github/workflows/scorecard.yaml:33]: Here, you're just installing `kind`

File: .github/workflows/scorecard.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        run: ./hack/install/install-kuttl.sh
+
+      - name: "start kind"
+        env:
+          KUBE_VERSION: ${{ matrix.kube-version }}
+        run: kind create cluster --config kind-$KUBE_VERSION.yaml
+
+      - name: "wait until cluster is ready"
+        run:  kubectl wait --timeout=5m --for=condition=available deployment/coredns -n kube-system
```
Comment (iblancasa) [inline .github/workflows/scorecard.yaml:42]: Is this really needed? When the `kind` command exists, the cluster should be up and running

File: Makefile [hunk 3]
```diff
@@ -377,3 +377,7 @@ cmctl:
 	mv $$TMP_DIR/cmctl $(CMCTL) ;\
 	rm -rf $$TMP_DIR ;\
 	}
+
+.PHONY: scorecard-tests
+scorecard-tests: operator-sdk
+	$(OPERATOR_SDK) scorecard -w=5m bundle || (echo "scorecard test failed" && exit 1)
```
Comment (iblancasa) [inline Makefile:383]: Missing new line at the end of the file

**Review by iblancasa — CHANGES_REQUESTED** [review]

File: .github/workflows/scorecard.yaml (hunk 2 above)
Comment (yuriolisa) [reply .github/workflows/scorecard.yaml:42]: Yes, sometimes the kind takes longer to be available.

File: .github/workflows/scorecard.yaml (hunk 1 above)
Comment (yuriolisa) [reply .github/workflows/scorecard.yaml:33]: Thank you, removing the wrong description.

**Review by iblancasa — APPROVED** [review]

**Comment by andreasgerstmayr** [comment]
Superseded by #353

**Pull request closed without merging**